galette/templates/default/elements/group.html.twig

```diff
@@ -5,12 +5,17 @@
 
 <div class="ui stackable pointing inverted menu tabbed">
     <div class="item header">
-        <i class="users cog icon" aria-hidden="true"></i>
-        {{ group.getFullName() }} :
+        <div class="ui inverted breadcrumb">
+            {% for parent in group.parents %}
+                <a class="section">{{ parent }}</a>
+                <i class="right angle icon divider"></i>
+            {% endfor %}
+            <a class="section active">{{ group.getName() }}</a>
```

Comment:
```suggestion
            <span class="section active">{{ group.getName() }}</span>
```

Reply:
The `active` class here has no effect because of the `item header` container on line 7 : the whole breadcrumb content is bold.